OSDOCS-9050 Security groups changes for OSD files (#69335)
openshift/openshift-docs: modules/osd-create-cluster-ccs.adoc

If you are installing a cluster into a Shared VPC, the VPC name and subnets are shared from the host project.
====
endif::osd-on-gcp[]
ifdef::osd-on-aws[]

> .. Optional: Expand *Additional security groups* and select additional custom security groups to apply to nodes in the machine pools created by default. You must have already created the security groups and associated them with the VPC you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.

Reviewer: ```suggestion
.. Optional: Expand *Additional security groups* and select additional custom security groups to apply to nodes in the machine pools that are created by default. You must have already created the security groups and associated them with the VPC that you selected for this cluster. You cannot add or edit security groups to the default machine pools after you create the cluster.
```
- From [IBM Style](https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences#grammar): "Include the word _that_ in clauses unless it results in writing that sounds too formal or stilted. The use of the conjunction _that_, although technically optional in some sentences, is never wrong and makes the sentence easier to translate and clearer for readers whose primary language is not English."

Author: Found here also: modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc

+

> By default, the security groups you specify will be added for all node types. Uncheck the *Apply the same security groups to all node types* checkbox to apply different security groups for each node type.

Reviewer: ```suggestion
By default, the security groups you specify are added for all node types. Clear the *Apply the same security groups to all node types* checkbox to apply different security groups for each node type.
```
- Avoid future tense where possible. IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense
- Use "clear" instead of "uncheck" for checkboxes. IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#checkbox

Author: here also: modules/rosa-sts-creating-a-cluster-with-customizations-ocm.adoc - changed here also

+

> For more information, see the requirements for _Security groups_ under _Additional resources_.

Reviewer: ```suggestion
For more information, see the requirements for security groups in the "Additional resources" section.
```
- See IBM Style: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting#highlighting-by-element
- No highlighting is needed for the phrase "security groups" in this sense. If you were referring to a title in the documentation, you could use double-quotation marks.
- I updated the highlighting of "Additional resources" from italics to double-quotation marks per the IBM Style "Highlighting" topic.
